packages/entity-renderer/renderer/common/shrink.js

```diff
@@ -1,8 +1,8 @@
-import { splitIfVocab } from "@zazuko/rdf-entity-webcomponent/src/builder/utils.js";
+import { splitIfVocab } from '@zazuko/rdf-entity-webcomponent/src/builder/utils.js'
 
-function shrink(urlStr) {
-  const { vocab, value } = splitIfVocab(urlStr);
-  return vocab ? `${vocab}:${value}` : `${value}`;
+function shrink (urlStr) {
+  const { vocab, value } = splitIfVocab(urlStr)
+  return vocab ? `${vocab}:${value}` : `${value}`
 }
 
-export { shrink };
+export { shrink }
```

Comment:
Curious how this differs from `shrink` in `@zazuko/prefixes`?

Reply:
Thanks for the suggestion. I will give it a try.
I think it's better if the one from `@zazuko/prefixes` can be used, so that we have a more consistent behavior across our apps.

Reply:
It seems to be working, so let's do the switch.